assets/js/blocks/cart/cart-cross-sells-product-list/cart-cross-sells-product.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import {
+	InnerBlockLayoutContextProvider,
+	ProductDataContextProvider,
+} from '@woocommerce/shared-context';
+import { ProductResponseItem } from '@woocommerce/type-defs/product-response';
+
+/**
+ * Internal dependencies
+ */
+import { Block as ProductImage } from '../../../atomic/blocks/product-elements/image/block';
+import { Block as ProductName } from '../../../atomic/blocks/product-elements/title/block';
+import { Block as ProductRating } from '../../../atomic/blocks/product-elements/rating/block';
+import { Block as ProductSaleBadge } from '../../../atomic/blocks/product-elements/sale-badge/block';
+import { Block as ProductPrice } from '../../../atomic/blocks/product-elements/price/block';
+import AddToCartButton from '../../../atomic/blocks/product-elements/add-to-cart/block';
+
+interface CrossSellsProductProps {
+	crossSellsProduct: ProductResponseItem;
```

Comment:
Maybe it would be better to call this `product`? Does prepending `crossSells` add any value?

Reply:
In all the examples you linked, the product is still named `product` - this one is the odd-one-out because it's called `crossSellsProduct`

Reply:
By the way I was referring to `crossSellsProduct` on line 21 here.

Reply:
Thanks for adding more context. I thought you were referring to line 20. I just implemented your suggestion.